### ISSUE TITLE
audiocache not working on my android simulator with android studio

### ISSUE THREAD

**Issue by susilsus** [luanpotter/audioplayers#601]
The usage of the code written is:
```

onPressed: () {
                  final player = AudioCache();
                  player.play('note1.wav');
                  //audioCache.play('assets/note1.wav');
                },
```

On execution no errors and app runs but while clicking the button, audio is not playing.
The error in the terminal is as follows:
     V/MediaPlayer(24905): resetDrmState:  mDrmInfo=null mDrmProvisioningThread=null mPrepareDrmInProgress=false 
    mActiveDrmScheme=false
    V/MediaPlayer(24905): cleanDrmObj: mDrmObj=null mDrmSessionId=null
What to do ?

**Comment by ahmedsaptan** [comment]
i have the same problem

**Comment by j3k007** [comment]
i am using this package in my android studio but when i compile my project, i am getting an error 
org-dartlang-app:/web_entrypoint.dart:9:8: Error: Error when reading 'lib/generated_plugin_registrant.dart': The system cannot find the file specified.

import 'package:xylophone/generated_plugin_registrant.dart';
       ^
org-dartlang-app:/web_entrypoint.dart:14:3: Error: Method not found: 'registerPlugins'.
  registerPlugins(webPluginRegistry);
  ^^^^^^^^^^^^^^^
Failed to compile application.

**Comment by luanpotter** [comment]
@susilsus that is not an error on the terminal, it's just regular logs from android audio that I was unable to get rid of. if you had no other errors on the console I will have to ask for the complete example code for me to repro, but it could be something on your setup (emulator volume, etc).

@j3k007 this error is something wrong with your setup and nothing to do with audioplayers, also, it's a completely unrelated issue so please don't piggyback on this issue.

**Comment by jeiselvan** [comment]
I have the same issue as susilsus. I cannot hear the sound in Android emulator. I've increased my volume to max and still no sound whatsoever.

**Comment by infinZero** [comment]
> The usage of the code written is:
> 
> ```
> 
> onPressed: () {
>                   final player = AudioCache();
>                   player.play('note1.wav');
>                   //audioCache.play('assets/note1.wav');
>                 },
> ```
> 
> On execution no errors and app runs but while clicking the button, audio is not playing.
> The error in the terminal is as follows:
> V/MediaPlayer(24905): resetDrmState: mDrmInfo=null mDrmProvisioningThread=null mPrepareDrmInProgress=false
> mActiveDrmScheme=false
> V/MediaPlayer(24905): cleanDrmObj: mDrmObj=null mDrmSessionId=null
> What to do ?

Same issue with same file name & formate , could anyone resolve this issue or we'll be remain waiting ???

**Comment by jeiselvan** [comment]
I think the issue is it is not working with the Android Emulator. Once running in a physical Android device, the sound works fine. In the iOS emulator the sounds are coming perfectly as well. I don't know what the issue is, but as long as it is working on a physical device I am fine with it.

**Comment by luanpotter** [comment]
Can you please try using the `example` app? It is working perfectly well for me including the android emulator. If the example app is working then compare your code because it's something wrong with your code. Otherwise must be some sound config wrong on your emulator.

**Comment by infinZero** [comment]
> I think the issue is it is not working with the Android Emulator. Once running in a physical Android device, the sound works fine. In the iOS emulator the sounds are coming perfectly as well. I don't know what the issue is, but as long as it is working on a physical device I am fine with it.

Yes you are right, after continuously trying to make correction and finding fault in my code, finally I tried it in iOS Simulator where it works fine.

**Comment by Jolium** [comment]
I "fixed" that problem sticking with version 0.16.1. I think it is a problem with version 0.16.2.

I just changed the pubspec.yaml:
audio players: ^ 0.16.1 <<< removed this
audio players: '0.16.1' <<< added this

Then I run flutter clean (not sure if it did anything) and get pub.

**Comment by erickzanardo** [comment]
There isn't enough info here for us to act on and has been inactive for some time, closing this for now, feel free to open a new issue with more info if the error persists.
